mc_test/mcTestCall.cpp

```diff
@@ -18,8 +18,8 @@
  *  Generates a TestCertificateProof;
  *
  *  3) ./mcTest "create" "csw" "darlin/cob_marlin" <"-v"> <"-zk"> "proof_path" "params_dir" "amount" "sc_id"
- *  "nullifier" "mc_pk_hash" "end_cum_comm_tree_root" "cert_data_hash",
- *  Generates a TestCSWProof.
+ *  "nullifier" "mc_pk_hash" "end_cum_comm_tree_root" <"cert_data_hash">,
+ *  Generates a TestCSWProof. cert_data_hash is optional
  */
 
 void create_verify_test_cert_proof(std::string ps_type_raw, int argc, char** argv) {
```

Comment:
nullifier field_t is not freed at the end of this function

Reply:
Done